setup.js

```diff
@@ -129,17 +129,27 @@ const setupCampaigns = () => {
 const setupValidationRules = () => {
   const rules = [
     {
-      name: "More-than-$55-in-Cart",
+      name: "More-than-$90-in-Cart-and-Pixel",
       rules: {
         1: {
           name: "order.amount",
           conditions: {
-            $more_than: [5500],
+            $more_than: [9000],
           },
           error: {
-            message: "Total cart value must be greater than $150",
+            message: "Total cart value must be greater than $88",
```

Comment:
I've done something different now: I've created separate file "voucherifySetup" where I will keep all the campaigns, rules, segments info with variables to create them for each testing session. Hopefully, it will make everything more consistent.